bin.src/validateDrp.py

```diff
@@ -73,6 +73,8 @@
                         help='Skip making plots of performance.')
     parser.add_argument('--level', type=str, default='design',
                         help='Level of SRD requirement to meet: "minimum", "design", "stretch"')
+    parser.add_argument('--skipGalaxies', dest='skipGalaxies', default=False, action='store_true',
```

Comment:
`--skipGalaxies` is likely ambiguous if you're just thinking about the KPMs.  You might be like "Well, duh, of course I don't want galaxies in my stellar photometry KPMs".

Is it actually necessary to have this option?  When computing things in {{calcnonsrc}}, can you just log a simple message that these metrics aren't available and then skip them?